spec/order_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'order'
+require 'customer'
+require 'nil'
+
+
+describe Order do
+  let(:order){Order.new}
+  let(:customer){double :Customer}
+  let(:customer_order){ order.select_items({"Duck Rice" => 8, "Gyoza" => 5, "Mango" => 3}) }
+  let(:small_order){ order.select_items({"Duck Rice" => 8}) }
+
```

Comment:
lines 9 and 10 do not really belong in let statements - writing a method would be better.